tests/testSBMLSuite.py

```diff
@@ -80,6 +80,8 @@ def test_sbml_testsuite_case(test_number, result_path):
 
         # simulate model
         rdata = amici.runAmiciSimulation(model, solver)
+        if rdata['status'] != amici.AMICI_SUCCESS:
+            raise amici.sbml_import.SBMLException('Simulation Failed')
```

Comment:
@FFroehlich : This may easily cover up future unexpected failures. What about introducing/handling more specific status codes for these event-related issues?

Reply:
That just moves the problem further down the road. Add explicit check for test id so this is only allowed for prespecified tests.

Reply:
Thanks. Works for me.